server/middlewares/validateID.js

```diff
@@ -0,0 +1,21 @@
+import partiesData from '../models/parties';
+
+const validateID = (req, res, next) => {
+  const id = Number(req.params.id);
+  if (Number.isNaN(id) || id % 1 !== 0 || id < 1 || String(req.params.id).indexOf('.') !== -1) {
```

Comment:
I don't think that the last 2 checks are necessary since they are basically checking for floating point numbers which has already been taken care of by the second check. What do you think?

Reply:
The second to the last check is not necessary as you pointed out, but the last check is actually checking for 1.000... or should that be valid?

Reply:
Alright. Thanks